### PR TITLE
Expanded IDE instructions for PHPStorm

### PR DESCRIPTION
**Pull request by photodude**

### CODE REVIEW

**Comment by aik099** [comment]
:+1:

File: IDE/readme.md [hunk 1]
```diff
@@ -5,10 +5,14 @@ Go to Preferences->PHP->Code style->Formatter and Import ```eclipse_pdt3_formatt
...
 * PHPStorm
-Copy the file ```joomla_phpstorm.xml``` to:
+1.Copy the file ```joomla_phpstorm.xml``` to:
  	- Mac ```~/Library/Preferences/WebIdeXX/config/codestyles/Joomla.xml```
  	- Windows ```<User home>\.WebIdeXX\config\codestyles\Joomla.xml```
  	- Linux ```~/.WebIdeXX/config/codestyles/Joomla.xml```
+2. Restart PHPStorm
+3. Open settings
+  - Editor > Code Style
+  - Manage the scheme and selet *Joomla* > *Copy to Project*
```
Comment (elkuku) [inline IDE/readme.md]: Small typo: sele**c**t

Otherwise: :+1:  :wink:

**Comment by photodude** [comment]
@elkuku Thanks for the catch, that typo is now corrected. 
I also fixed some of the formatting

**Comment by photodude** [comment]
Reference for this change
https://www.jetbrains.com/phpstorm/help/customizing-profiles.html

**Comment by rvbgnu** [comment]
Hello,
I was trying to make a PR against the repos, but did not work. But only updating your PR will be enough to change the link to PhpStorm help page:
See 7ca5ee5e21951a26527504457fb4acf86cea95fb

**Comment by photodude** [comment]
@rvbgnu I have corrected the link in the PR as requested.

**Comment by photodude** [comment]
@mbabker ping for merge consideration

**Pull request merged**